dist/cjs/trustedtypes.api_only.js

```diff
@@ -278,6 +434,91 @@ const trustedTypesBuilderTestOnly = function() {
     return exposedPolicies.get(pName) || null;
   }
 
+  /**
+   * Returns the name of the trusted type required for a given element
+   *   attribute.
+   * @param {string} tagName The name of the tag of the element.
+   * @param {string} attribute The name of the attribute.
+   * @param {string=} elementNs Element namespace.
+   * @param {string=} attributeNs The attribute namespace.
+   * @return {string|undefined} Required type name or undefined, if a Trusted
+   *   Type is not required.
+   */
+  function getAttributeType(tagName, attribute, elementNs = '',
+      attributeNs = '') {
+    // TODO: Support namespaces.
+    const canonicalAttr = toLowerCase.apply(String(attribute));
+    return getTypeInternal_(tagName, 'attributes', canonicalAttr);
+  }
+
+  /**
+   * Returns a type name from a type map.
+   * @param {string} tag A tag name.
+   * @param {string} container 'attributes' or 'properties'
+   * @param {string} property The attribute / property name.
+   * @return {string|undefined}
+   * @private
+   */
+  function getTypeInternal_(tag, container, property) {
+    if (HTML_TYPE_MAP['*'][container][property]) {
+      return HTML_TYPE_MAP['*'][container][property];
+    }
```

Comment:
I see that '*' has little on it so this seems ok as written.

But here, the general rule trumps a specific rule which is the reverse of what we've done for things like https://github.com/Polymer/polymer-resin/blob/master/lib/contracts/contracts.js which has a general rule for `*[src]` -> TrustedScriptURL and specific ones for `img[src], video[src], audio[src]` -> TrustedURL.

Is there a maintenance hazard due to that?

Reply:
I guess we can revert this in code (first look in the element, then in '*'). For current values this doesn't seem to cause any  behavior change. FWIW `*[src]` in not a Trusted Type at all, we only cover `src` for selected elements (e.g. because of Web Components that may create their own attributes with arbitrary semantics).